### PR TITLE
gui/settings dialog: Fix refresh option stretch the display area.

### PR DESCRIPTION
**Pull request by Zangetsu38**
# About
- Fix allow Apply change stretch display area state in real time.
Fix refresh module list when open setting dialog.

### CODE REVIEW

File: vita3k/gui/src/settings_dialog.cpp [hunk 1]
```diff
@@ -228,6 +229,8 @@ void init_config(GuiState &gui, EmuEnvState &emuenv, const std::string &app_path
         config.ngs_enable = emuenv.cfg.ngs_enable;
         config.psn_status = emuenv.cfg.psn_status;
     }
+    get_modules_list(gui, emuenv);
```
Comment (Macdu) [inline vita3k/gui/src/settings_dialog.cpp:232]: What does this have to do with the PR?

File: vita3k/renderer/include/renderer/state.h [hunk 2]
```diff
@@ -89,8 +89,8 @@ struct State {
     void set_surface_sync_state(bool disable) {
         disable_surface_sync = disable;
     }
-    void set_stretch_display(bool disable) {
-        stretch_the_display_area = disable;
+    void set_stretch_display(bool state) {
+        stretch_the_display_area = state;
```
Comment (Macdu) [inline vita3k/renderer/include/renderer/state.h:93]: Give it a meaningful name like stretch (also in the header) and not state

File: vita3k/gui/src/settings_dialog.cpp (hunk 1 above)
Comment (Zangetsu38) [reply vita3k/gui/src/settings_dialog.cpp:232]: Is fix

File: vita3k/renderer/include/renderer/state.h (hunk 2 above)
Comment (Zangetsu38) [reply vita3k/renderer/include/renderer/state.h:93]: State is perfect, why stretch ? Is strange
Is bool indicate litteraly state ?

Comment (Macdu) [reply vita3k/renderer/include/renderer/state.h:93]: state is perfect? If I call set_stretch_display with state = false, are you able to deduce what it does? Does it stretch the display or not?

Comment (Zangetsu38) [reply vita3k/renderer/include/renderer/state.h:93]: I no understund ? 
State false = disable ? So is clear 🤔 I no understund what you wabt said here 🤷‍♂️

Comment (Zangetsu38) [reply vita3k/renderer/include/renderer/state.h]: done

**Review by Macdu — APPROVED** [review]
